docs/services/metadata/example.json

```diff
@@ -0,0 +1,271 @@
+{
```

Comment:
Do we need this file anymore if there is `api-example.puml` which I think is 1:1 with this one?